### ISSUE TITLE
"Run Interact" 

### ISSUE THREAD

**Issue by lheagy** [geoscixyz/em-apps#22]
In the plots that we manually update, the button that does the update says "Run Interact", but all of our instructions say that the button says "Run Plot". This has the potential to cause confusion. 

So either, the instructions should be updated to say "Run Interact" or the button updated to say "Run Plot" 

cc @thast

**Comment by thast** [comment]
yes, this is due to the change in ipywidgets again. Before the manual button was taking the name of the function, while now it has a default name 'run interact'.
My opinion is that the default is fine. We should update the instructions.

**Comment by lheagy** [comment]
kk, sounds good. Could you guys come up with a game-plan for updating those?

**Comment by thast** [comment]
I can update the DC ones now. Should not take long.
I do not know if others apps uses the manual option.

**Comment by thast** [comment]
#23

**Comment by lheagy** [comment]
@thast: can we close this?
